### ISSUE TITLE
Discover Tribes Tab/UI in App

### ISSUE THREAD

**Issue by jimBeaux27** [stakwork/sphinx-ios#108]
Create a simple way for users to discover new tribes/content with a Discover/Explore tab that shows a UI similar to this https://community.sphinx.chat/t. Some points
1. User selects discover tab
2. We present a UI similar to this https://community.sphinx.chat/t
3. When the user selects the tab, they can get basic info about the tribe
4. They can click a UIButton that lets them enter the tribe

I think this would make the app stickier and more personalized because people could discover new tribes more easily

**Comment by pitoi** [comment]
Agreed it's time to integrate listings tribes in the app. We kept it separate to allow for multiple tribe servers/directories. 
We'll need to make sure the implementation supports merging multiple tribe servers

**Comment by jimBeaux27** [comment]
Good idea Paul we should have a sound spec for this and think about the what ifs.

One idea I had was what if we implement the above link but with a WebView. It would at least make it easily accessible to the app audience, it would be fast to ship and potentially low technical debt.

**Comment by pitoi** [comment]
Agreed on the webview, maybe a simple selector at the top that is active when the user wants to visit a different tribe directory. There is a similar options under the profile to set the meme-server for file storage.

**Comment by jimBeaux27** [comment]
I'd argue the discover feature would be critical for onboarding a "normie". In my opinion, we should be dumping them in the discover tab and tell them to find tribes immediately after sign up. We need to get them from account to stimulating content as quickly as possible.
